azure-pipelines.yml

```diff
@@ -205,6 +205,37 @@ stages:
               STORAGE_API_TOKEN_READ_ONLY: $(STORAGE_API_TOKEN_READ_ONLY)
               RUN_SYNAPSE_TESTS: $(RUN_SYNAPSE_TESTS)
 
+      - job: gcsTests
+        displayName: 'GCS Tests'
+        steps:
+          - script: |
+              docker-compose run -e STORAGE_API_URL=$STORAGE_API_URL_CONTROL_PLANE -e STORAGE_API_TOKEN=$STORAGE_API_TOKEN_CONTROL_PLANE tests ./vendor/bin/phpunit --testsuite gcs-tests --debug
```

Comment:
`STORAGE_API_TOKEN_CONTROL_PLANE` zadanej v pipelines mám a stejně mi to v testech tvrdí `401 Unauthorized`

Počítám že v `env:` sekci to být nemusí. `STORAGE_API_URL_CONTROL_PLANE` to schroupne a volá to správně control plane stack.

Reply:
To je právě to, že to v env sekci být musí, protože je to secret a ten se tam sám nepředá

Reply:
zkusim si to zapamatovat :)

Reply:
diky